client/components/themes-list/style.scss

```
@@ -17,3 +17,39 @@
 	margin: 0 10px;
 	flex-grow: 1;
 }
+
+.themes-list__empty-container {
+	.themes-list__not-found-text {
+		color: var(--studio-gray-60);
+		margin-top: 40px;
+		margin-bottom: 20px;
+	}
+
+	.themes-list__upgrade-section-wrapper {
+		display: flex;
+		flex-direction: column;
+		align-items: center;
+		padding-top: 64px;
+		background: #fff;
+		box-shadow: 0 0 0 1px var(--color-border-subtle);
+	}
+
+	.themes-list__upgrade-section-title {
+		font-size: $font-title-small;
+		color: var(--studio-gray-100);
+	}
+
+	.themes-list__upgrade-section-subtitle {
+		width: 680px;
```

This fixes the small overflow when resizing the screen and provides a bit of padding to the text for small screens
```suggestion
		max-width: 680px;
	   padding: 0 2rem;
```

---

Fixed at https://github.com/Automattic/wp-calypso/pull/68542/commits/cd3857d37651e679d2aa4e45b8a615ed5fc26835